maths/monte_carlo_integration_univariate.py

```diff
@@ -0,0 +1,146 @@
+"""
+This program finds the approximate integration value/area under the curve of
+a specified function within specified limits using Monte Carlo integration method.
+
+Further, a graph of the individual areas under the curve considered for the calculation
+is also plotted. (PLOT SECTION -> Optional implementation)
+
+Link to source material:
+https://towardsdatascience.com/the-basics-of-monte-carlo-integration-5fe16b40482d
+"""
+
+import doctest
+import random
+
+import numpy as np
+
+
+def function_to_be_integrated(univariate_variable: float) -> float:
+    """
+    Function to calculate the sin of a particular value of x
+
+    >>> round(function_to_be_integrated(0))
+    0
+    """
+    return np.sin(univariate_variable)
+
+
+# declaring list to store all the values for plotting, globally
+plt_vals: list[float] = []
+
+
+def monte_carlo(
+    lower_limit: float, upper_limit: float, number_of_sections: int
+) -> float:
+    """
+    Monte Carlo integration function (Doctests written w.r.t sin(x))
+
+    >>> round(monte_carlo(0, np.pi, 1000))
+    2
+    >>> round(monte_carlo(0, np.pi, 10))
+    2
+    >>> round(monte_carlo(0, 2*np.pi, 1000))
+    0
+    >>> round(monte_carlo(-2*np.pi, 2*np.pi, 1000))
+    0
+    >>> round(monte_carlo(-10.2345678, 1.89712, 1000))
+    0
+    """
```

Comment:
You haven't added exceptional cases where the function would raise Value error for garbage input as @cclauss mentioned. Add those invalid cases as well.